website/pages/docs/platform/k8s/helm/examples/enterprise-dr-with-raft.mdx

```diff
@@ -0,0 +1,193 @@
+kubectl exec -ti vault-primary-0 -- vault operator raft list-peers
+
+Node                                    Address                        State       Voter
+----                                    -------                        -----       -----
+a1799962-8711-7f28-23f0-cea05c8a527d    vault-primary-0.vault-primary-internal:8201    leader      true
+e6876c97-aaaa-a92e-b99a-0aafab105745    vault-primary-1.vault-primary-internal:8201    follower    true
+4b5d7383-ff31-44df-e008-6a606828823b    vault-primary-2.vault-primary-internal:8201    follower    true
+```
+
+## Secondary Cluster
+
+With the primary cluster created, next create a secondary cluster and enable 
+disaster recovery replication.
+
+```shell
+helm install vault-secondary \
+  --set='server.image.repository=hashicorp/vault-enterprise' \
+  --set='server.image.tag=1.4.0-rc1_ent' \
+  --set='server.ha.enabled=true' \
+  --set='server.ha.raft.enabled=true' \
+  https://github.com/hashicorp/vault-helm/archive/v0.5.0.tar.gz
+```
+
+Next, initialize and unseal `vault-secondary-0` pod:
+
+```shell
+kubectl exec -ti vault-secondary-0 -- vault operator init
+kubectl exec -ti vault-secondary-0 -- vault operator unseal
+```
+
+Finally, join the remaining pods to the Raft cluster and unseal them.  The pods
+will need to communicate directly so we'll configure the pods to use the internal
+service provided by the Helm chart:
+
+```shell
+kubectl exec -ti vault-secondary-1 -- vault operator raft join http://vault-secondary-0.vault-secondary-internal:8200
+kubectl exec -ti vault-secondary-1 -- vault operator unseal
+
+kubectl exec -ti vault-secondary-2 -- vault operator raft join http://vault-secondary-0.vault-secondary-internal:8200
+kubectl exec -ti vault-secondary-2 -- vault operator unseal
+```
+
+To verify if the Raft cluster has successfully been initialized, run the following.
+
+First, login using the `root` token on the `vault-secondary-0` pod:
+
+```shell
+kubectl exec -ti vault-secondary-0 -- vault login
+```
+
+Next, list all the raft peers:
+
+```shell
+kubectl exec -ti vault-secondary-0 -- vault operator raft list-peers
+
+Node                                    Address                        State       Voter
+----                                    -------                        -----       -----
+a1799962-8711-7f28-23f0-cea05c8a527d    vault-secondary-0.vault-secondary-internal:8201    leader      true
+e6876c97-aaaa-a92e-b99a-0aafab105745    vault-secondary-1.vault-secondary-internal:8201    follower    true
+4b5d7383-ff31-44df-e008-6a606828823b    vault-secondary-2.vault-secondary-internal:8201    follower    true
+```
+
+## Enable Disaster Recovery Replication On Primary
+
+With the initial clusters setup, we can now configure them for disaster recovery replication.
+
+First, on the primary cluster, enable replication:
+
+```shell
+kubectl exec -ti vault-primary-0 -- vault write -f sys/replication/dr/primary/enable primary_cluster_addr=https://vault-primary-active:8201
```

Comment:
Dumb question, but is replication always over `https`? Just noticing that `http` was used above for the `raft join` commands.

Reply:
Yeah it is, I think Vault auto upgrades it.  I'll test and double check.